ci-operator/config/openshift/jenkins/master.json

```diff
@@ -0,0 +1,172 @@
+{
+    "tag_specification": {
+        "cluster": "https://api.ci.openshift.org",
+        "namespace": "openshift",
+        "name": "jenkins",
+        "tag": "",
+        "tag_overrides": {}
+    },
+    "base_images": {
+        "base": {
+            "cluster": "https://api.ci.openshift.org",
+            "namespace": "openshift",
+            "name": "origin-v3.11",
+            "tag": "base"
+        },
+        "cluster-tests": {
+            "cluster": "https://api.ci.openshift.org",
+            "namespace": "openshift",
+            "name": "origin-v3.11",
+            "tag": "tests"
+        }
+    },
+    "test_base_image": {
+        "cluster": "https://api.ci.openshift.org",
+        "namespace": "openshift",
+        "name": "release",
+        "tag": "golang-1.9"
```

Comment:
You're golang based?

Reply:
We are language agnostic.  No binaries of any sort are compiled.  It is unclear to me why we need a `test_base_image` at all for our scenario, but excluding this section fails `ci-operator`'s dependency check for this file.